index.html

```diff
@@ -61,8 +74,32 @@ <h1>Badge</h1>
 
     <!-- Badge on an Avatar -->
     <span class="badge">
-        <img src="/Assets/developer.png" class="avatar-img" alt="" srcset=""> <span class="badge-status">🔴</span>
+        <img src="/Assets/developer.png" class="avatar avatar-sm" alt="" srcset=""> <span class="badge-status">🔴</span>
     </span>
+
+    <hr>
+
+        <!-- Buttons -->
+        <h1>Buttons</h1>
+        <button class="btn">Default Button</button>
+        <button class="btn btn-primary">Primary Button</button>
+        <button class="btn btn-secondary">Secondary button</button>
+        <button class="btn btn-primary-outline">Primary Outline</button>
+        <button class="btn btn-secondary-outline">Secondary Outline</button>
+        <button class="btn btn-primary" ><a class="btn-link" href="#">link</a></button>
```

Comment:
Here you can remove the button tag and design the a tag however you want

Reply:
Thankyou for the code review , I will surely do the changes you suggested.